cmd/sync.go

```diff
@@ -153,6 +153,9 @@ func (raw *rawSyncCmdArgs) cook() (cookedSyncCmdArgs, error) {
 	}
 
 	cooked.followSymlinks = raw.followSymlinks
+	if err = crossValidateSymlinksAndPermissions(cooked.followSymlinks, true /* replace with real value when available */); err != nil {
```

Comment:
>true [](start = 69, length = 4)

Shouldn't this be false, right now, if sync doesn't support persist smb permissions?

Reply:
Sync also doesn't support follow symlinks right now.  

So, by hard coding true, here, it will immediately fail when someone first makes sync support symlinks, and then they'll know to hook in the smp property correctly.

I.e. anything we hard-code here is incorrect, but better to have something that will obviously fail when symlink support is first tested (by us, not by customer!) than something which will silently do the wrong thing, and therefore never get fixed.

Reply:
Ah, I suppose that does make sense. I thought we did have symlinks enabled in sync. Perhaps you should add a comment here describing that.